api/README.md

```diff
@@ -102,6 +102,17 @@ oc process -f api/openshift/templates/deploy.yaml \
 | FLYWAY_IMAGE_NAME          | The URL for the NATS service.
 | FLYWAY_IMAGE_TAG          | The Flyway source image tag that will be deployed
 
+The following configuration points are used to provide links from a project's details to the namespace the project uses.
+| Name                                | Description |
+| :------------------------------- | :-----------|
+| PRODUCTION_NAMESPACE_GOLD_URL    | The URL to the Production Namespace in the Gold Cluster.
```

Comment:
Hey @ShovelHand, thanks for your work on this! I noticed that there are a lot of environment variables that we are adding to the project in this PR. I was wondering if we could potentially cut down on the number of environment variables by defining a base URL and then concatenating the terms Test, tools, production, development, gold and silver to the base url. Of course, this would only be possible if part of the url for each of those is common. It's just an idea, I'm not fully convinced that this is better but its just something to consider

Reply:
That's a great idea! I'll do that.